DiagnosticsExtension/Models/ConnectionStringValidator/Constants.cs

```diff
@@ -14,38 +14,51 @@ namespace DiagnosticsExtension.Models.ConnectionStringValidator
+        public const string AuthFailureQueueStorageDocs = "See See <a href= 'https://docs.microsoft.com/azure/azure-functions/functions-bindings-storage-queue-trigger#connection-string' target='_blank'>relevant docs</a>.";
+        public const string AuthFailureFileShareStorageDocs = "See See <a href= 'https://docs.microsoft.com/azure/storage/common/storage-account-keys-manage' target='_blank'>relevant docs</a>.";
+        public const string AuthFailureServiceBusDocs = "See See <a href= 'https://docs.microsoft.com/azure/azure-functions/functions-bindings-service-bus-trigger#connection-string' target='_blank'>relevant docs</a>.";
+        public const string AuthFailureEventHubsDocs = "See See <a href= 'https://docs.microsoft.com/azure/azure-functions/functions-bindings-event-hubs-trigger#connection-string' target='_blank'>relevant docs</a>.";
+        public const string StorageAccessRestrictedDetails = "This can be due to firewall rules on the resource.  Please check if you have configured firewall rules or a private endpoint and that they correctly allow access from the Function App.  See <a href= 'https://docs.microsoft.com/en-us/azure/storage/common/storage-network-security?tabs=azure-portal' target='_blank'>Storage account network security</a> for additional details.";
+        public const string ServiceBusAccessRestrictedDetails = "This can be due to firewall rules on the resource.  Please check if you have configured firewall rules or a private endpoint and that they correctly allow access from the Function App.  See <a href= 'https://docs.microsoft.com/en-us/azure/service-bus-messaging/network-security' target='_blank'>Service Bus network security</a> for additional details.";
+        public const string EventHubAccessRestrictedDetails = "This can be due to firewall rules on the resource.  Please check if you have configured firewall rules or a private endpoint and that they correctly allow access from the Function App.  See <a href= 'https://docs.microsoft.com/en-us/azure/event-hubs/network-security' target='_blank'>Event Hubs network security</a> for additional details.";
+        public const string DnsLookupFailed = "The service resource specified in the connection string was not found. Please check the value of the setting.";
+        public const string FQNamespaceResourceNotFound = "The resource specified in the app setting '{0}" + FullyQualifiedNamespace + "' was not found. Please check the value of the setting.";
+        public const string StorageAccountResourceNotFound = "The resource specified in the app setting '{0}' was not found. Please check the value of the setting.";
+        public const string MalformedConnectionStringDetails = "The connection string configured in app setting '{0}' is invalid (e.g.missing some required elements).";
```

Comment:
```suggestion
        public const string MalformedConnectionStringDetails = "The connection string configured in app setting '{0}' is invalid. Please double check your connection string";
```

I think it's better to not go into 'missing some required elements' since a customer might not necessarily know what it means to have the correct elements of a connection string

Reply:
Changed.